react/src/lib/components/DeckGLMap/layers/LassoLayer/lassoLayer.stories.tsx

```diff
@@ -0,0 +1,182 @@
+            }
+            setAlignment(newAlignment);
+            if (lassoLayer[0].visible) {
+                setArgsState(enableLassoArgs);
+            } else {
+                setArgsState(disableLassoArgs);
+            }
+        },
+        [argsState]
+    );
+
+    React.useEffect(() => {
+        setEditedData(args.editedData);
+    }, [args.editedData]);
+
+    return (
+        <>
+            <div className={useStyles().main}>
+                <DeckGLMap
+                    id={"DeckGL-Map"}
+                    {...argsState}
+                    editedData={editedData}
+                    setProps={(updatedProps) => {
+                        setEditedData(updatedProps);
+                    }}
+                />
+            </div>
+            <ToggleButtonGroup
+                value={alignment}
+                exclusive
+                onChange={handleChange1}
+                aria-label="text alignment"
+            >
+                <ToggleButton value="left" aria-label="left aligned">
+                    <LayersIcon />
+                </ToggleButton>
+            </ToggleButtonGroup>
+        </>
+    );
+};
+
+const enableLassoArgs = {
+    id: "DeckGL-Map",
+    resources: {
+        propertyMap:
+            "https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/propertyMap.png",
+        depthMap:
+            "https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/propertyMap.png",
+        wellsData:
+            "https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/volve_wells.json",
+        logData:
+            "https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/volve_logs.json",
+    },
+    bounds: [432205, 6475078, 437720, 6481113],
+    layers: [
+        {
+            "@@type": "WellsLayer",
+            data: "@@#resources.wellsData",
+            logData: "@@#resources.logData",
+            logrunName: "BLOCKING",
+            logName: "ZONELOG",
+            logColor: "Stratigraphy",
+        },
+        {
+            "@@type": "PieChartLayer",
+            data: "https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/piechart.json",
+        },
+        {
+            "@@type": "LassoLayer",
+            visible: true,
+            data: "@@#resources.wellsData",
+            logData: "@@#resources.logData",
+            logrunName: "BLOCKING",
+            logName: "ZONELOG",
+            logColor: "Stratigraphy",
+        },
+    ],
+    editedData: {},
+    views: {
+        layout: [1, 1],
+        showLabel: false,
+        viewports: [
+            {
+                id: "view_1",
+                show3D: false,
+                layerIds: [],
+            },
+        ],
+    },
+};
+
+const disableLassoArgs = {
```

Comment:
Please consolidate `enable/disableLassoArgs`. They are almost identical.

Also, please remove props that are not relevant to this example.

Also, please use local resource urls (see other examples).